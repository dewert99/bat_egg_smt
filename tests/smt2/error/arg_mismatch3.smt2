(declare-fun f (Bool) Bool)
(assert f)