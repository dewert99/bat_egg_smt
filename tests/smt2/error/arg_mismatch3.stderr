the function `f` expects 1 arguments but got 0 at 2:9
(assert f)
        ^

